taskt/Core/Automation/Commands/DataTable Commands/GetDataRowValueCommand.cs

```diff
@@ -1,67 +1,70 @@
 ﻿using System;
-using System.Linq;
-using System.Xml.Serialization;
+using System.Collections.Generic;
 using System.Data;
+using System.Linq;
 using System.Windows.Forms;
-using System.Collections.Generic;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("DataTable Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to get a DataRow Value from a DataTable")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to add a datarow to a DataTable.")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("DataTable Commands")]
+    [Description("This command gets a DataRow Value from a DataTable.")]
+
     public class GetDataRowValueCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the DataRow Name")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter a existing DataRow to get Values from.")]
-        [Attributes.PropertyAttributes.SampleUsage("**myData**")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_DataRowName { get; set; }
+        [PropertyDescription("DataRow")]
+        [InputSpecification("Enter an existing DataRow to get Values from.")]
+        [SampleUsage("{vDataRow}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        public string v_DataRow { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Select value by Index or Column Name")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Index")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Column Name")]
-        [Attributes.PropertyAttributes.InputSpecification("Select whether the DataRow value should be found by index or column name")]
-        [Attributes.PropertyAttributes.SampleUsage("Select from **Index** or **Column Name**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Search Option")]
+        [PropertyUISelectionOption("Column Name")]
+        [PropertyUISelectionOption("Index")]
```

Comment:
Although it looks good but I think you should use **"Column Index"** for better understanding or clarity.

Reply:
done

Reply:
Also update its references on line #62 and #83.